modules/rules_closure/0.15.0/presubmit.yml

```diff
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+    - macos
```

Comment:
nit: add macos_arm64 and ubuntu2204, and maybe windows? We can also do it in the next version.